diff --git a/device/queueconstants_android.go b/device/queueconstants_android.go
index 88662c222d6c70c5cfc52fbece117844f8310e3f..bacc6a5e1706a45ba66e0a81d538359b1656fd82 100644
--- a/device/queueconstants_android.go
+++ b/device/queueconstants_android.go
@@ -17,3 +17,3 @@ const (
 	MaxSegmentSize             = (1 << 16) - 1 // largest possible UDP datagram
-	PreallocatedBuffersPerPool = 4096
+	PreallocatedBuffersPerPool = 1024
 )
