diff --git a/build.bat b/build.bat
--- a/build.bat
+++ b/build.bat
@@ -23 +23 @@
-	call :download wireguard-tools.zip https://git.zx2c4.com/wireguard-tools/snapshot/wireguard-tools-06a99cce2c9998f53eb30d2f258a9e5ff286445b.zip b7a73e027cee3127f3cccba8ad3a08ea61ccd42d3ea5c28c548a8e0ec9e10cf6 "--exclude wg-quick --strip-components 1" || goto :error
+	call :download wireguard-tools.zip https://codeload.github.com/WireGuard/wireguard-tools/zip/06a99cce2c9998f53eb30d2f258a9e5ff286445b 209db11b588eb4dc55a05ee70ceea44690ddad44e945d8299e95465a5dec4d7d "--exclude wg-quick --strip-components 1" || goto :error
